osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs

```diff
@@ -25,12 +26,16 @@ public class LegacyOldStyleSpinner : CompositeDrawable
         private Sprite metreSprite;
         private Container metre;
 
+        private bool spinnerNoBlink;
```

Comment:
Given that this is always used in a `!spinnerNoBlink` manner, I'd like to see this variable inverted as `spinnerBlink`. You can do the inverse locally - don't need to change the name of the setting enum.